.github/workflows/ci.yml

```diff
@@ -78,13 +88,19 @@ jobs:
 
     - name: Test docstring examples with doctest
       if: matrix.python-version == '3.10'
-      run: pytest src/ README.rst
+      run: coverage run --data-file=.coverage-doctest --module pytest src/ README.rst
+
+    - name: Coverage report for doctest only
+      if: matrix.python-version == '3.10'
+      run: |
+        coverage report --data-file=.coverage-doctest
+        coverage xml --data-file=.coverage-doctest -o doctest-coverage.xml
```

Comment:
Rename the report for doctest as it is being run over the local files, but the tests are covering the installed library. So just highlighting that the reports should be different.